src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2635,9 +2641,7 @@ namespace NServiceBus.Routing
     public class UnicastRoutingTable
     {
         public UnicastRoutingTable() { }
-        public void RouteTo(System.Type messageType, NServiceBus.Routing.IUnicastRoute route) { }
-        public void RouteTo(System.Type messageType, NServiceBus.Routing.IUnicastRoute route, bool overrideExistingRoute) { }
-        public void SetFallbackRoute(System.Func<System.Type, NServiceBus.Extensibility.ContextBag, System.Threading.Tasks.Task<NServiceBus.Routing.IUnicastRoute>> fallbackRoute) { }
+        public void AddOrReplaceRoutes(object sourceKey, System.Collections.Generic.IList<NServiceBus.Routing.RouteTableEntry> entries) { }
```

Comment:
this will change our proposed approach to get schema support into sql transport. Quickly discussed this with @SzymonPobiega and we think there are still simple enough ways to support it but we should allocate some time to help sql transport to implement it.